webextensions/chrome/background.js

```diff
@@ -299,25 +306,44 @@ const ThinBridgeTalkClient = {
     }
     console.log(`* Result: [${matchedSectionNames.join(', ')}]`);
 
-    if (redirectCount == 0) {
-      console.log(`* No redirection: fallback to default`);
-      if (config.DefaultBrowser == '' ||
-          String(config.DefaultBrowser).toLowerCase() == BROWSER.toLowerCase()) {
-        console.log(`* Continue to load as the default reaction`);
-        loadCount++;
-      }
-      else {
-        console.log(`* Redirect to the default browser ${config.DefaultBrowser}`);
-        redirectCount++;
+    if (isActionMode) {
+      // a.k.a "full mode" in IE View WE
+      console.log(`* Dispatch as action mode`);
+      if (redirectCount > 0 || loadCount == 0) {
+        console.log(`* Redirect to another browser`);
+        this.redirect(url, tabId, closeTabCount > 0);
       }
+      console.log(`* Continue to load: ${loadCount > 0}`);
+      return loadCount == 0;
     }
+    else {
+      // Compatible mode with ManifestV2 version of this add-on
+      console.log(`* Dispatch as compatible mode`);
+
+      if (loadCount > 0) {
+        console.log(`* Continue to load`);
+        return false;
+      }
 
-    if (redirectCount > 0 || loadCount == 0) {
-      console.log(`* Redirect to another browser`);
-      this.redirect(url, tabId, closeTabCount > 0);
+      if (redirectCount > 0) {
+        console.log(`* Redirect to another browser`);
+        this.redirect(url, tabId, closeTabCount > 0);
+        return true;
+      }
+
+      if (config.DefaultBrowser) {
+        console.log(`* Use DefaultBrowser: ${config.DefaultBrowser}`);
+        if (String(config.DefaultBrowser).toLowerCase() == BROWSER.toLowerCase()) {
+          return false;
+        } else {
+          this.redirect(url, tabId, closeTabCount > 0);
+          return true;
+        }
+      } else {
+        console.log(`* DefaultBrowser is blank`);
+        return false;
+      }
```

Comment:
This block intend to keep same logic with MV2 version's one:
https://github.com/ThinBridge/ThinBridge/blob/973debb5d97d63d40b42ab98eb0b4d710ff04902/webextension/edge/background.js#L243-L251

Reply:
I've verified that this branch passes all tests added by #66.

Reply:
Thanks!